packages/core/apps/core-apps-server-internal/index.ts

```diff
@@ -6,4 +6,4 @@
  * Side Public License, v 1.
  */
 
-export { CoreApp } from './core_app';
+export { CoreAppsService } from './src';
```

Comment:
Changed the service class name to keep consistent with the browser-side service class counterpart.